src/javascript/_common/analytics.js

```diff
@@ -1,11 +1,43 @@
 const DerivAnalytics = require('@deriv-com/analytics');
+const Cookies = require('js-cookie');
+const { getCountry } = require('./utility');
+const LocalStore = require('./storage').LocalStore;
+const Language = require('./language');
+const { getAppId } = require('../config');
 
 const Analytics = (() => {
-    const init = () => {
+    const init = async () => {
+        const account_type = LocalStore?.get('active_loginid')
```

Comment:
<img width="593" alt="Screenshot 2024-11-06 at 4 06 28 PM" src="https://github.com/user-attachments/assets/209e6017-b8ca-45a1-be2c-72e271128ffe">

you dont need to do this.

client.accounts has something called broker for each account object

Reply:
updated. please review